src/resources/postcss/tickets-registration-page.pcss

```diff
@@ -179,7 +186,7 @@ body.page-tribe-attendee-registration {
 				-ms-grid-columns: 6fr 1fr 2fr 2fr;
 				grid-template-columns: 6fr 1fr 2fr 2fr;
 
-				&.tribe-tickets__item--price-sufix {
+				&.tribe-tickets__item--price-suffix {
```

Comment:
If we change this here, we break the old view which has the mispelling "sufix" in the class name there.

Reply:
No, actually we saved the v1 version as `tickets-v1.pcss` so we are free to do this and anything we want. Also, eventually change class names (Something I thought of, in order to make the structure more BEM friendly). 

https://github.com/moderntribe/event-tickets/blob/bucket/ari-js-normalization/src/resources/postcss/tickets-v1.pcss